aspnet/tutorials/publish-to-azure-webapp-using-vs.rst

```diff
@@ -1,47 +1,181 @@
 
-5. On the **Connection** tab of the **Publish Web** dialog box, click **Publish**.
+In the **New ASP.NET Core Web Application (.NET Core)** dialog:
 
-.. image:: publish-to-azure-webapp-using-vs/_static/05-PublishWebdb.png
+- Tap **Web Application**
+- Verify **Authentication** is set to **Individual User Accounts**
+- Verify **Host in the cloud** is **not** checked
+- Tap **OK**
 
-You can view the publishing progress in the **Web Publish Activity** window within Visual Studio.
+.. image:: publish-to-azure-webapp-using-vs/_static/noath.png
 
-.. image:: publish-to-azure-webapp-using-vs/_static/06-WebPublishActivityWindow.png
+Test the app locally
+^^^^^^^^^^^^^^^^^^^^^
 
-When publishing to Azure is complete, your web app will be displayed in a browser running on Azure. 
+- Use **Ctl-F5** to run the app in non-debug mode
+- Tap the **About** and **Contact** links. Depending on the size of your device, , you might need to click the navigation icon to show the links
 
-.. image:: publish-to-azure-webapp-using-vs/_static/07-Browser.png
+.. image:: publish-to-azure-webapp-using-vs/_static/show.png
 
+- Tap **Register** and register. You can use a fictitious email. When you submit, you'll get the following error:
 
+.. image:: publish-to-azure-webapp-using-vs/_static/mig.png
+
+You can fix the problem two different ways:
+
+- Tap **Apply Migrations** and when the page updates, refresh the screen
+- Run the following from the command line in the project directory::
+
+    dotnet ef database update
+
+The app displays the email registration used to register and a **Log off** link.
+
+
+.. image:: publish-to-azure-webapp-using-vs/_static/hello.png
+
+Deploy the app to Azure
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Right click on the project and select **Publish**
+
+.. image:: publish-to-azure-webapp-using-vs/_static/pub.png
+
+In the **Publish** dialog, tap **Microsoft Azure App Service**
+
+.. image:: publish-to-azure-webapp-using-vs/_static/maas1.png
+
+Tap **New** to enter a new resource group. Creating a new resource group will make it easy to delete all the Azure resources you create in this tutorial.
+
+.. image:: publish-to-azure-webapp-using-vs/_static/newrg1.png
+
+Create a new resource group and app service plan:
+
+- Tap **New** for the resource group and enter a name for the new resource group
+- Tap **New** for the  app service plan and select a location near you. You can keep the default generated name
+- Tap **Explore additional Azure services** to create a new database
+
+.. image:: publish-to-azure-webapp-using-vs/_static/cas.png
+
+- Tap the green **+** icon to create a new SQL Database
+
+.. image:: publish-to-azure-webapp-using-vs/_static/sql.png
+
+- Tap  **New** on the **Configure SQL Database** dialog
+
+.. image:: publish-to-azure-webapp-using-vs/_static/conf.png
+
+- You can keep the default **Server Name**
+- Enter an administrator user name and password, and then tap **OK**
```

Comment:
I'm not sure but I think once this is set it cannot be retrieved in the portal, if so then maybe it makes sense to tell users to make sure to remember what they entered.